azext_iot/sdk/iothub/service/models/configuration_content_py3.py

```diff
@@ -21,16 +21,20 @@ class ConfigurationContent(Model):
     :type modules_content: dict[str, dict[str, object]]
     :param module_content: The module configuration content.
     :type module_content: dict[str, object]
+    :param modules_purchase: The modules purchase content.
+    :type modules_purchase: dict[str, dict[str, object]]
     """
 
     _attribute_map = {
         'device_content': {'key': 'deviceContent', 'type': '{object}'},
         'modules_content': {'key': 'modulesContent', 'type': '{{object}}'},
         'module_content': {'key': 'moduleContent', 'type': '{object}'},
+        'modules_purchase': {'key': 'modulesPurchase', 'type': '{{object}}'},
```

Comment:
Add a comment with an identifier/detail that this is a custom change to autogenerated content.